cmd/Main.hs

```diff
@@ -45,20 +47,31 @@ import qualified Web.OpenWeatherMap.Types.Wind as Wind
 appName :: String
 appName = "openweathermap"
 
+parseAction :: Parser (Either Client.Location String)
+parseAction = (Left <$> parseLocation ) <|> (Right <$> strOption
```

Comment:
I don't get it (right/left), but:

1. This is not an action.
2. I think there would be just one option added to the existing: `-f, --forecast`. In the future there could be another like `-p, --past` for historical data.

Reply:
So it maybe more sensible to make a custom sumtype for this rather than using Either? Eg:

```haskell
data Endpoint = Forecast String
                        | Current Location
```